### DISCUSSION TITLE
[图片显示效果优化] 建议优化图片显示方式

### DISCUSSION THREAD

**Discussion by wxyhgk** [tangly1024/NotionNext#2023] [Ideas]
我这个截图效果就很不错,有立体感,有阴影

<img width="1202" alt="image" src="https://user-images.githubusercontent.com/37659181/204129549-c97703e3-7e3d-4c01-ba83-8ba7f2d50700.png">

博客原始的图片显示效果太过于平面,显示效果不好

**Comment by tangly1024** [comment]
不错的提议👍
